Scripts/SetAndHandleEmpty/SetAndHandleEmpty.yml

```diff
@@ -0,0 +1,41 @@
+commonfields:
+  id: SetAndHandleEmpty
+  version: -1
+name: SetAndHandleEmpty
+fromversion: 4.5.0
+type: python
+tags:
+- Utility
+comment: Checks if value exists in context. If the value exists it will be set in context,
+  if not nothing will be set.
+enabled: true
+args:
+- name: key
+  required: true
+  default: true
+  description: The key to set
+- name: value
+  description: The value to set to the key. Can be an array. Usually, a dq expression.
```

Comment:
The value of the key to set in context. Can be an array. The value is usually a DQ expression.